.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "submodules/shiny"]
 	path = submodules/shiny
 	url = https://github.com/vimc/montagu-shiny
+[submodule "montagu-bb8"]
```

Comment:
Is there a reason, aside from `backup` being at the top level, not to put this under `submodules`?

Reply:
I feel like the things in `submodules` are the things that we just include as submodules so we can do git magic and track which docker images to deploy. Whereas `backup` and `bb8` are different in that we are actually using the contents of the submodules to do things. So I wanted to keep them separate. I agree that a folder named `submodules` is not exactly the clearest way to do that, but moving submodules around is a pig.

Reply:
yeah, that makes sense